client/src/Components/button.js

```diff
@@ -2,24 +2,20 @@ import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
 
-/*function searchCorrectHandling() {
-  const [product, setProduct] = useState()
-
-function getCorrectHandling () {
-  const correctHandling = useEffect(() => {
-    localStorage.getItem('product')
-  })
-}
-on Click={searchCorrectHandling}
-*/
-
-
-
 const SearchButton = () => {
   return (
-    <div className="container">
       <button>Search</button>
-    </div>
   );
 };
+
+// const ButtonStyled = styled.button`
```

Comment:
same

Reply:
these lines are actually kept for the button, which will now follow. If u don't mind I would leave these settings for now and delete them once and for all, when my button is ready for the page routings.